label_studio/core/settings/base.py

```diff
@@ -56,6 +56,10 @@
             'handlers': ['console'],
             # 'propagate': True,
         },
+        'core': {
```

Comment:
why do we need this?

Reply:
I added this because in the`InactivitySessionTimeoutMiddleware` the following is being logged:

https://github.com/HumanSignal/label-studio/blob/2e360bc0087aef02a6f1b2ca5e0e4eeec7698202/label_studio/core/middleware.py#L216-L219

But this was not visible in the console. It would have saved me a lot of time to figure out why I couldn't login into the Django admin, so that's why I added it.

As an alternative, I now changed the above logging to `logger.warn`: https://github.com/HumanSignal/label-studio/pull/6800/commits/70b0b978122fd928b3f4233737d0005e46edaeb5

Now it is default shown in the console without adding a separate logger config for the `core` app:
```
[2024-12-18 09:15:47,382] [core.middleware::process_request::217] [WARNING] Request is too far from last login 1734513347 > 1209600; logout
```